### DISCUSSION TITLE
Cave green blobs deal burn damage instead of acid burn

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14211] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The cave objects known as `mushroomcavebulb1/2/3/4` deal burn affliction instead of acid burn.

Honestly not a big deal but feels like they should have been updated

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/f1be3a14-3fbf-4389-89ec-278586854265)

### Version

v1.5.8.0 (Summer Update)

**Comment by JeroonBosch** [comment]
Thank you for your report, good find. Issue has been addressed internally, so I'll close this as completed.
